README.md

```diff
@@ -61,7 +61,10 @@
 
 
 - **[Navbar]**-> a navigation bar.
-- **[header]**-> general introduction to the programmer
+- **[header]**-> general introduction to the programmer.
+- **[Body]**-> general introduction to the projects done by programmer.
```

Comment:
- [ Optional ]  good Job by doing the last reviewer required comments but it would be better to fix the margin and the padding here its causing some problems 
![image](https://user-images.githubusercontent.com/74744787/212492175-cc484094-f899-4ae9-b3a2-98f54d5a42b9.png)